py-deploy/tldeploy/cli.py

```diff
@@ -99,9 +99,10 @@ def cli(ctx, version):
 @currency_network_contract_name_option
 @click.option(
     "--expiration-time",
-    help="Expiration time of the currency network after which it will be frozen",
+    help="Expiration time of the currency network after which it will be frozen (disabled by default)",
```

Comment:
I would instead write `(0 means disabled)` since we show a default of 0 in the tool.

Reply:
:heavy_check_mark: